mat-process-api.Tests/V1/Controllers/ProcessDataImagesControllerTests.cs

```diff
@@ -301,21 +296,20 @@ public void given_an_unexpected_error_would_be_thrown_when_GetProcessImage_contr
                     return new GetProcessImageResponse(null, DateTime.MinValue, null); //dummy return that will never happen due to error being thrown prior to it. It's only needed to keep compiler happy.
                 });
 
-            var request = new GetProcessImageRequest();
             _mockGetValidator.Setup(x => x.Validate(It.IsAny<GetProcessImageRequest>())).Returns(new FV.ValidationResult()); //validation successful
 
             //act
-            var controllerResponse = _processImageController.GetProcessImage(request);
+            var controllerResponse = _processImageController.GetProcessImage(new GetProcessImageRequest());
             var result = controllerResponse as ObjectResult;
 
             //assert
             Assert.NotNull(controllerResponse);
             Assert.NotNull(result);
-            Assert.AreEqual(500, result.StatusCode);
+            Assert.AreEqual(expectedStatusCode, result.StatusCode);
         }
 
         [Test]
-        public void given_an_unexpected_error_would_be_thrown_when_GetProcessImage_controller_method_is_called_then_it_returns_a_correct_error_message()
+        public void given_an_unexpected_error_would_be_thrown_when_GetProcessImage_controller_method_is_called_then_it_returns_a_correctly_formatted_error_message_that_is_based_of_exception()
```

Comment:
You shouldn't use 'Returns' and generate an error. Use .Throws<> instead for when mocking the use case in this test.

Reply:
I wasn't happy with that implementation either. I'll try and see if I can make throws work with exceptions helper. 

But reviewing your PR gave me an idea of rewriting helper to return errors rather than throw them. Somehow it never registered to me that you can put an exception into a variable :D

Reply:
Alright, I managed to get this working. Tests are much much simpler to follow now. Thanks for the suggestion!